packages/app/src/app/pages/Dashboard/index.tsx

```diff
@@ -169,7 +170,7 @@ export const Dashboard: FunctionComponent = () => {
               as="main"
               css={css({
                 width: '100%',
-                height: 'calc(100vh - 48px)',
+                height: `calc(100% + ${GUTTER}px)`,
```

Comment:
This change breaks the scroll on the recent page:

![qgi7kv-3000 preview csb app_dashboard_recent_workspace=411542e0-c5fb-4ee8-85c9-65c8cfd18e52](https://user-images.githubusercontent.com/24959348/216207163-a7819a4e-988a-4e5f-b98b-f977fcbf2e84.png)

It's not a problem on the other pages that are using `react-virtualized` and only render a list of items. But the recent page has various sections and the page doesn't grow to fit them after this change.